assets/js/admin/meta-boxes-order.js

```diff
@@ -851,46 +857,49 @@ jQuery( function ( $ ) {
 			}
 		},
 
-		bulk_actions: {
-
-			check_column: function() {
-				if ( $( this ).is( ':checked' ) ) {
-					$( '#woocommerce-order-items' ).find( '.check-column input' ).attr( 'checked', 'checked' );
-				} else {
-					$( '#woocommerce-order-items' ).find( '.check-column input' ).removeAttr( 'checked' );
-				}
-			},
+		select_row: function() {
+			var $row = false;
+			if ( $( this ).is( 'tr' ) ) {
+				$row = $( this );
+			} else {
+				$row = $( this ).closest('tr');
```

Comment:
There are a few lines like this where there should be spacing between the parentheses and arguments.